rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -39,6 +40,8 @@ static int wait_timeout;
 bool debugging = false;
 bool is_oot_driver;
 bool backend_kvm = false;
+bool tls_server = false;
```

Comment:
TLS_SERVER is only used to control the global flag tls_server.

```C
#ifdef TLS_SERVER
bool tls_server = true;
#else
bool tls_server = false;
#endif
```

So this sort of codes can be removed:

```C
#ifdef TLS_SERVER
if (tls_server) {
  ...
}
```